pylint/checkers/stdlib.py

```diff
@@ -184,11 +192,20 @@ class StdlibChecker(BaseChecker):
 
     def _check_bad_thread_instantiation(self, node):
         if not node.kwargs and node.args:
-            self.add_message('bad-thread-instantiation', node=node)
+            self.add_message(
```

Comment:
What's wrong with the previous version?

Reply:
Sorry, it was just accidentally reformated, i will revert it to original